config/crd/bases/infrastructure.cluster.x-k8s.io_azuremanagedmachinepools.yaml

```diff
@@ -199,6 +199,13 @@ spec:
             description: AzureManagedMachinePoolSpec defines the desired state of
               AzureManagedMachinePool.
             properties:
+              additionalTags:
```

Comment:
Is this file autogenerated or it is correct to update this yaml file ?

Reply:
It is auto generated. If you update it causing the autogen to give the same output -- the ci won't complain though.

But as a best practice it should be generated.
See `make generate` command in makefile.